tracks/r/exercises/anagram/mentoring.md

```diff
@@ -0,0 +1,64 @@
+### Reasonable solution
+
+This solution is clear and straight-forward.
+
+```r
+anagram <- function(subject, candidates) {
+  
+  subject_letters <- tolower(strsplit(subject, "")[[1]])
+  target <- sort(subject_letters)
+
+  is_anagram <- function(word) {
+    word_letters <- tolower(strsplit(word, "")[[1]])
+    identical(sort(word_letters), target) && !identical(word_letters, subject_letters)
+  }
+  
+  matches <- sapply(candidates, is_anagram)
+
+  if(!any(matches)) {
+    c()
+  } else {
+    candidates[matches]
+  }
+}
+```
+
+The learning goal here is common string operations and checking conditions on whole objects vs. on a per item level.
```

Comment:
Do "objects" & "item" here mean strings & character? If yes, how about being that specific?

Reply:
Done so. I do think the concept behind checking conditionals on a non-vector vs. a vector could be more generalized.